package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "humanize-duration-ts",
+  "version": "1.0.0",
```

Comment:
Can you please remove the version . I use 'semantic-release' for generating the version.